Jenkinsfile

```diff
@@ -64,5 +64,11 @@ pipeline {
                 archiveArtifacts artifacts: "$TEST_PROJECT_REPO_NAME/test_bundle.zip", onlyIfSuccessful: true
             }
         }
+
+        stage('trigger run on aws device farm') {
```

Comment:
You don't need to split every command into a stage. Installing requirements and running the tests make sense as one unit of work.

Reply:
done